Topic 0/src/com/globant/topic0/exercise3/builders/PrivateHospitalBuilder.java

```diff
@@ -0,0 +1,31 @@
+package com.globant.topic0.exercise3.builders;
+
+import com.globant.topic0.exercise3.model.Hospital;
+import com.globant.topic0.exercise3.model.PrivateHospital;
+
+// Concrete Builder, is responsible for creating and assembling a Hospital object
+
+public class PrivateHospitalBuilder extends HospitalBuilder {
+	
+	private PrivateHospital privateHospital;
+	
+	public PrivateHospitalBuilder(){
+		privateHospital = new PrivateHospital();
+	}
+	
+	@Override
+	public Hospital getHospital() {
+		return privateHospital;
+	}
+
+	@Override
+	public void buildRooms() {
```

Comment:
Same here for the pull request #2 , the 20 is hardcoded, and need to be parametrized argument.